tests/Dockerfile

```diff
@@ -1,15 +1,20 @@
 ARG PYTHON_VERSION=3.7
-
 FROM python:${PYTHON_VERSION}
 
 ARG APT_MIRROR
 RUN sed -ri "s/(httpredir|deb).debian.org/${APT_MIRROR:-deb.debian.org}/g" /etc/apt/sources.list \
     && sed -ri "s/(security).debian.org/${APT_MIRROR:-security.debian.org}/g" /etc/apt/sources.list
 
-RUN apt-get update && apt-get -y install \
+RUN apt-get update && apt-get -y install --no-install-recommends \
     gnupg2 \
     pass \
-    curl
+    curl \
```

Comment:
I think that `curl` and `ssh` are already in the Python 3.7 Debian image:
```
docker run --rm python:3.7 sh -c "which curl && which ssh"
/usr/bin/curl
/usr/bin/ssh
```

If there's another reason you need them, I prefer having lists like this sorted so that it's easy to check if something's being installed.